### ISSUE TITLE
BUG: Mailbox Statistics - Request failed with status code 500

### ISSUE THREAD

**Issue by kfear27** [KelvinTegelaar/CIPP#741]
When opening mailbox statistics and selecting a tenant an error is thrown:
Request failed with status code 500

![image](https://user-images.githubusercontent.com/54527192/153119070-8f1f3e92-a061-4730-bd9f-314e06e7741d.png)

Email and Exchange > Reports > Mailbox Statistics

This happens to all tenants

I have updated to the latest version and the issue still persists.
A permissions check returns that the app has all the permissions it requires.

**Comment by KelvinTegelaar** [comment]
not a bug. your exchange token is not working.

**Comment by kfear27** [comment]
I managed to refresh the token successfully and update, it still failed. I wiped the resource group in Azure, along with the forked repo's. I then reconfigured the entire thing with a fresh app, permissions, tokens etc. Yet I am still receiving the same errors.

When I run the token permissions test script I see multiple
Connect-graphAPI : Could not log into the Graph API for tenant tenant-id-here: The remote server returned an error: (400) Bad Request

How can this fail with all new tokens?
The installation guide was followed to a tee.
